pylint/lint/run.py

```diff
@@ -243,6 +270,7 @@ def __init__(
                         "callback": Run._not_implemented_callback,
                         "help": "Load and enable all available extensions. "
                         "Use --list-extensions to see a list all available extensions.",
+                        "hide_from_config_file": True,
```

Comment:
I'm not sure if we actually support this. I'll open a follow-up issue to investigate.

Either way, I don't think we should add it in the default output of this flag as it can only confuse users.

Reply:
What I was sawing is we should probably hide rc-file from the config file too ? Or did I misunderstand the purpose of "hide_from_config_file" ?